src/components/Comments/Comment.js

```diff
@@ -52,7 +52,8 @@ const Comment = ({
     upvoteRemoved: null,
     downvoteInserted: null,
     downvoteRemoved: null
-  }
+  },
+  refreshComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refreshComment` prop is a good change as it allows the component to refresh comments when necessary. This can be useful in scenarios where the comment data changes and the UI needs to reflect these changes.
```diff
+  refreshComment
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->